src/io/flutter/project/FlutterProjectOpenProcessor.java

```diff
@@ -55,7 +46,7 @@ public Icon getIcon() {
   @Override
   public boolean canOpenProject(@Nullable VirtualFile file) {
     if (file == null) return false;
-    final PubRoot root = PubRoot.forDirectoryWithRefresh(file);
+    final PubRoot root = PubRoot.forDirectory(file);
```

Comment:
I forgot the exact error but we can't refresh here. Since the user is looking at the project open dialog, presumably IDEA's filesystem cache is pretty up to date.